scheduling/shortest_job_first.py

```diff
@@ -126,15 +126,20 @@ def calculate_average_times(
     for i in range(no_of_processes):
         print("Enter the arrival time and brust time for process:--" + str(i + 1))
         arrival_time[i], burst_time[i] = map(int, input().split())
-    waiting_time = calculate_waitingtime(arrival_time, burst_time, no_of_processes)
+
+    waiting_time = calculate_waitingtime(
+        arrival_time, burst_time, no_of_processes)
+
     bt = burst_time
     n = no_of_processes
     wt = waiting_time
     turn_around_time = calculate_turnaroundtime(bt, n, wt)
+
     calculate_average_times(waiting_time, turn_around_time, no_of_processes)
-    processes = list(range(1, no_of_processes + 1))
```

Comment:
I do not understand this change.

Reply:
processes is redefined exactly the same way afterwards
without new user input